product_docs/docs/tde/15/troubleshooting.mdx

```diff
@@ -1,39 +1,61 @@
 ---
 title: "Troubleshooting with encrypted WAL files"
 navTitle: WAL files
+deepToC: true
 ---
 
-You can encrypt WAL files. When troubleshooting with encrypted WAL falls, you can use WAL command options. 
+When TDE is enabled, WAL files are encrypted. If you want to perform operations on the encrypted WAL files, you need to allow the operations to decrypt the file. 
+
+When troubleshooting with encrypted WAL files, you can use WAL command options. 
 
 ## Dumping a TDE-encrypted WAL file
 
-To work with an encrypted WAL file, the [pg_waldump](https://www.postgresql.org/docs/15/pgwaldump.html) needs to be aware of the unwrap key. You can either pass the key for the unwrap command using the following options to the `pg_waldump` command or depend on the fallback environment variable:
+To work with an encrypted WAL file, you need to ensure the [pg_waldump](https://www.postgresql.org/docs/current/pgwaldump.html) utility can access the unwrap key and decrypt it. For this purpose, the utility requires awareness of three values.
+
+Pass these values using the following options to the `pg_waldump` command.
 
 ### `--data-encryption`
 
-Consider the WAL files to encrypt, and decrypt them before processing them. You must specify this option if the WAL files were encrypted by transparent data encryption. `pg_waldump` can't automatically detect whether WAL files are encrypted. Optionally, specify an AES key length. Valid values are 128 and 256. The default is 128.
+Specify this option if the WAL files were encrypted by transparent data encryption.
+
+The `--data-encryption` or `-y` option ensures the command is aware of the encryption. Otherwise, `pg_waldump` can't detect whether WAL files are encrypted.
+
+It considers the WAL files to encrypt and decrypts them before processing them. Optionally, specify an AES key length. Valid values are `128` and `256`. The default is `128`.
```

Comment:
I couldn't make out what this means. What does "consider" mean in this context. Does it evaluate them for some purpose?

Reply:
Thanks for pointing this out, as I had forgotten to rephrase this part. WDYT? 

```suggestion
Provide the same encryption configuration you used when initializing the TDE-enabled database cluster. For example, if you specified an AES key length during the cluster creation, you must specify it here as well. Otherwise, run the flag with no values.
```